package.json

```diff
@@ -0,0 +1,62 @@
+{
+  "name": "resin-procbots",
+  "version": "0.0.1",
+  "description": "Process robots for automating CI pipelines",
+  "homepage": "https://github.com/resin-io-modules/resin-procbots#readme",
+  "preferGlobal": true,
+  "types": "lib/**/*.ts",
```

Comment:
Agree, and it's what I did for using `resin-discoverable-services`. I'm not sure why I've done this.